hydrus/data/resource_based_classes.py

```diff
@@ -34,6 +36,31 @@ def get_type(object_: Dict[str, Any]) -> str:
     return type_
 
 
+def get_modified_object(object_, doc, path):
+    temp_object = copy.deepcopy(object_)
+    class_ = doc.parsed_classes[path]
+    properties = class_["class"].supportedProperty
+    datetimefields = []
+    for prop in properties:
+        kwargs = getattr(prop, 'kwargs', None)
+        range = kwargs.get('range')
+        title = getattr(prop, 'title')
+        if range is not None:
+            if "dateTime" in range:
+                datetimefield = title
+                datetimefields.append(datetimefield)
+    if len(datetimefields) != 0:
+        for field in datetimefields:
+            try:
+                datetime_value = temp_object.get(field)
+                dt_object = datetime.strptime(datetime_value, "%d/%m/%Y %H:%M:%S")
```

Comment:
Please use standard tools to check for string compliance. See [this answer](https://stackoverflow.com/a/61569783/2536357)